app/views/pages/cookies.html.erb

```diff
@@ -0,0 +1,161 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      Cookies
+    </h1>
+
+    <p class="govuk-body-l">
+      TDLR about cookies
+    </p>
+
+    <p class="govuk-body">
+    Cookies are used to:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>measure how you use the this service so it can be updated and improved based on your needs</li>
+      <li>remember the notifications you've seen so that we don't show them to you again</li>
+      <li>remember who you are when you have logged in so we can show you your information and information relevant to you</li>
```

Comment:
I don't think we do any of these things?

Reply:
Def agree bullet 3 should be removed. I think we will be tracking usage when in Beta using something like GA to inform us of bad pages. I think we will also use a cookie to say if they have seen the cookie banner before so.